jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -189,7 +247,11 @@ class Status extends React.Component {
           content: app.units.size()
         }, {
           columnSize: 2,
-          content: charm.path()
+          content: (
+            <span className="status-view__link"
+              onClick={this._navigateToCharm.bind(this, charmId)}>
+              {charm.path()}
```

Comment:
A funky thing happens where the revision is removed so it's not displayed and we have to generate the path again to get the string without the revision.